src/envs/__init__.py

```diff
@@ -17,6 +17,7 @@
     "CoverEnv",
     "CoverEnvTypedOptions",
     "CoverEnvHierarchicalTypes",
+    "CoverEnvNonrefinable",
```

Comment:
annoying suggestion: this env is not actually non-refinable, what we mean is non-downward-refinable. Can we rename to CoverEnvNondownwardref or maybe just CoverEnvRegrasp?